js/src/ext/analytics.js

```diff
@@ -7,9 +7,11 @@ Wu.Analytics = Wu.Class.extend({
 		// create shortcut
 		app.log = this.fire;
 
-		// initialize google analytics
-		ga('create', app.options.ga.id, 'auto');
-		ga('send', 'pageview');
+		if(!window.testMode){
```

Comment:
perhaps a simpler way to solve this problem, is to create a wrapper for `ga()`: 

1. First, rename the `ga` global object to `analytics` [like this](https://developers.google.com/analytics/devguides/collection/analyticsjs/renaming-the-ga-object)
2. Then create a wrapper function:
```javascript
function ga () {
    if (window.testMode) return;
    analytics(arguments); // or pass all five arguments one by one, if necessary
}
```
3. Then use this wrapper function like we're doing already: `ga('send', 'pageview')`